posthog/utils.py

```diff
@@ -544,6 +545,37 @@ def is_plugin_server_alive() -> bool:
         return False
 
 
+def is_clickhouse_alive() -> bool:
+    from posthog.client import sync_execute
+
+    try:
+        sync_execute("SELECT 1")
+        return True
+    except BaseException:
+        return False
+
+
+def is_kafka_alive() -> bool:
+    from ee.kafka_client.client import can_connect
+
+    return can_connect()
+
+
+def is_event_service_alive() -> bool:
+    absolute_url = absolute_uri("/e/")
+
+    # Somewhat confusing, but we don't want to make a valid
+    # request to the event service, because we don't want to
+    # accidentally create an event. So we just send a HEAD
+    # request (which will fail because there's no body) and
+    # verify the event service properly failed on the response.
+    try:
+        r = requests.head(absolute_url)
+        return r.status_code == 400
+    except BaseException:
+        return False
+
```

Comment:
@fuziontech thoughts on if these seem like appropriate ways to check if these services are alive?

Reply:
This is fantastic. Clickhouse and Kafka checks are perfect.

For events service use `/_health` as the endpoint (it's what we use as the health check in production)

Reply:
Great, just updated it to be the new endpoint. Thanks!

Reply:
@rcmarron take a look at https://github.com/PostHog/posthog/pull/8497 to avoid code duplication